superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -195,6 +195,50 @@ export const visTypes = {
     },
   },
 
+  time_pivot: {
+    label: t('Time Series - Period Pivot'),
```

Comment:
I think the terminology for this visualization is confusing, you intermix "period" and "frequency" a lot and I think both of those are confusing for people not already familiar with more complex data analysis.

What do you think of calling this `Time Series -- Seasonality Pivot` and using `seasonality` consistently? If you don't like that, I think `periodicity`  is next best.

cc @elibrumbaugh @graceguo-supercat for thoughts!

Reply:
Very good point. We're planning to use this mostly on periodicity of week and day at Lyft, so I prefer "periodicity" to "seasonality" which seems to imply year...

Reply:
I agree, "seasonality" does seem most associated with year although it doesn't have to be. "periodicity"++